File: examples/pascal/constructs/for_loop.pas
// Ported from examples/lumen/constructs/for_loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
var i: integer;
begin
    i := 0;
    while i < 10 do begin
        write(i);
        if i < 9 then begin
            write(', ');
        end;
        i := i + 1;
    end;
    writeln('');
end.

File: examples/php/constructs/for_loop.php
<?php
// Ported from examples/lumen/constructs/for_loop.lm by scripts/port_examples.py; edit the Lumen original, not this file.
$i = 0;
while ($i < 10) {
    print($i);
    if ($i < 9) {
        print(", ");
    }
    $i = $i + 1;
}
print("\n");
